bubuku/features/restart_if_dead.py

```diff
@@ -23,9 +24,13 @@ def run(self, current_actions) -> bool:
         if self.broker.is_running_and_registered():
             return False
         _LOG.info('Waiting for complete death')
-        self.broker.stop_kafka_process()
+        if not self.stopped:
+            if not self.broker.stop_kafka_process():
+                return True
+            self.stopped = True
         _LOG.info('Starting up again')
-        self.broker.start_kafka_process(self.zk.exhibitor.zookeeper_hosts + self.zk.prefix)
+        if not self.broker.start_kafka_process(self.zk.exhibitor.zookeeper_hosts + self.zk.prefix):
+            return True
         return False
```

Comment:
nope, at the very beginning i thought to include additional checks there, but then removed it.